linux

```diff
@@ -40,8 +40,9 @@ python:3.8 python-3.8
 python:3.9 python-3.9
 python:3.10 python-3.10
 python:3.11 python-3.11
+python:3.12 python-3.12
 ruby
 ruby:3.3 ruby-3.3
 swift
 ubuntu linux
-sbtscala/scala-sbt:eclipse-temurin-jammy-22_36_1.10.0_3.4.2 sbt1.10.0-scala3.4.2
\ No newline at end of file
+sbtscala/scala-sbt:eclipse-temurin-jammy-22_36_1.10.0_3.4.2 sbt1.10.0-scala3.4.2
```

Comment:
Issue: I don't think this was intentional to be re added

Reply:
This is not my change and I was also surprised to see this. Could we remove this, please?

Reply:
No, wasn't